regression/scripts/airfoil_analysis/airfoil_panel_method_test.py

```diff
@@ -0,0 +1,209 @@
+# airfoil_panel_method_test.py
+# 
+# Created:  
+# Modified: Mar 2021, M. Clarke 
+
+#----------------------------------------------------------------------
+#   Imports
+# ---------------------------------------------------------------------
+
+import SUAVE 
+import os
+from SUAVE.Core import Units, Data 
+from SUAVE.Methods.Aerodynamics.Airfoil_Panel_Method.airfoil_analysis import airfoil_analysis
+import matplotlib.pyplot as plt   
+from SUAVE.Methods.Geometry.Two_Dimensional.Cross_Section.Airfoil.compute_naca_4series \
+     import  compute_naca_4series
+from SUAVE.Plots.Airfoil_Plots  import plot_airfoil_properties
+
+import numpy as np
+
+# ----------------------------------------------------------------------
+#   Main
+# ----------------------------------------------------------------------
+
+def main():    
+    # Define Panelization 
+    npanel = 160
+    
+    # Define Reynolds Number
+    Re     = 5E6 
+    
+    #Define Angle of Attack
+    AoA1   = 3*Units.degrees 
+    AoA2   = 10*Units.degrees 
+    
+    # -----------------------------------------------
+    # SUAVE
+    # -----------------------------------------------
+    # Generate Airfoil Geometry (NACA 3310) 
+    airfoil_geometry   = compute_naca_4series(0.03,0.3,0.1,npoints=npanel )
+    
+    # Compute Airfoil Aerodynamic and Boundary Layer Properties 
+    airfoil_properties_1 = airfoil_analysis(airfoil_geometry,AoA1,Re, npanel) 
+    airfoil_properties_2 = airfoil_analysis(airfoil_geometry,AoA2,Re, npanel)
+    
+    # Plot Results 
+    plot_airfoil_properties(airfoil_properties_1,line_style='k-',arrow_color = 'r',plot_pressure_vectors = True)  
+    plot_airfoil_properties(airfoil_properties_2,line_style='b-',arrow_color = 'g',plot_pressure_vectors = True)  
+    
+    
+    # -----------------------------------------------
+    # XFOIL
```

Comment:
Cool, is this actually XFOIL data we're regressing with?

Reply:
Yes